react/features/base/i18n/dateUtil.ts

```diff
@@ -94,10 +92,13 @@ export function getLocalizedDurationFormatter(duration: number) {
     // showing the hour and we want to include the hour if the conference is
     // more than an hour long
 
+    // @ts-ignore
```

Comment:
Isn't there a types package for moment?

Reply:
There is. The problem is with `moment-duration-format` (which has types). Some extra config, or code changes might be needed for this to work